src/components/summary/study-info-summary.tsx

```
@@ -159,17 +170,102 @@ export default function SummaryStudyInfo({ data }: Props) {
 
     return '신청하기';
   };
 
-  const handleApplyClick = () => {
-    if (!isLoggedIn) {
-      router.push('/login');
+  const handlePaidStudyApplyClick = () => {
+    if (isVerificationLoading) return;
+    if (!isVerified) {
+      setIsPhoneVerificationOpen(true);
+      return;
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)